docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -11,26 +11,25 @@ include::_attributes.adoc[]
 :keywords: cors,http
 :extensions: io.quarkus:quarkus-vertx-http
 
-Cross-origin resource sharing (CORS) is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) other than its own from which a browser should permit loading resources.
+Cross-origin resource sharing (CORS) is an HTTP header system that enables a server to specify which external origins, comprising different domains, schemes, or ports, are allowed by a web browser to access its resources.
 
 [[cors-filter]]
 == CORS filter
 
-Quarkus provides a CORS filter which implements the `jakarta.servlet.Filter` interface and intercepts all incoming HTTP
-requests. It can be enabled in the Quarkus configuration file, `src/main/resources/application.properties`:
+Quarkus provides a CORS filter, which implements the `jakarta.servlet.Filter` interface and intercepts all incoming HTTP requests.
+It can be enabled in the Quarkus configuration file, `src/main/resources/application.properties`:
 
 [source, properties]
 ----
 quarkus.http.cors=true
 ----
 
-If the filter is enabled and an HTTP request is identified as cross-origin, the CORS policy and headers defined using the
-following properties will be applied before passing the request on to its actual target (servlet, Jakarta REST resource, etc.):
+When the filter is activated and identifies an HTTP request as cross-origin, it will enforce the CORS policy.
```

Comment:
Either "enabled" or "active". "Activated" sounds like it can be activated/deactivated dynamically, but this is something that happens just once for the whole application lifecycle.